python/lsst/obs/hsc/hscPupil.py

```diff
@@ -84,7 +84,7 @@ def _horizonRotAngle(self):
         alt = self.visitInfo.getBoresightAzAlt().getLatitude()
 
         # parallactic angle
-        sinParAng = (np.cos(lat.asRadians()) * np.sin(ha.asRadians()) /
+        sinParAng = (np.cos(lat.asRadians()) * np.sin(ha.asRadians()) / # noqa
```

Comment:
Why do you need an linting exception here?  You could do it on two lines, but I think the line break after binary operator linting rule has been put in as a permanent exception, so I don't think it will trigger travis even if you take it out.